.github/workflows/industrial_ci_action.yaml

```diff
@@ -0,0 +1,85 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - ros2
+      - noetic-devel
+      - melodic-devel
```

Comment:
The ROS1 branches will not work in ROS2 environments and vice versa.
```suggestion
```

Reply:
pushed rebases fixing this on each of the PRs